_contentTemplates/common/get-started.md

```diff
@@ -74,15 +73,9 @@ If you don't have an active license, [start a UI for Blazor trial](https://www.t
 #get-access
 ### Enable the Components in the Project
 
-To have the project use the Telerik UI for Blazor components, follow these steps:
```

Comment:
This is the 3rd time we ask the reader to start a free trial and download the components. Do we need it?

Reply:
Added to reduce confusion and avoid tickets. I recommend putting it back in. Trial uses should not have to wait a few days for us to tell them "read this article to get the feed running".